templates/default-layout.lucius

```diff
@@ -23,6 +25,16 @@
  **     <div .my-component__item>
  **
  ** Components can be nested.
+ **
+ **
+ ** TYPOGRAPHY
+ **
+ ** Typographic choices for sizes, line-heights and margins are based on a
+ ** musical major third scale (4:5). This gives us a way to find numbers
+ ** and relationships between them that are perceived as harmonic.
+ **
+ ** To make use of this modular scale, follow this formula:
+ **   (5/4)^n | n ∈ [-6, 8]
```

Comment:
I just wanted to check about the notation, since n ∈ [a, b] usually means that n is a real number such that a <= n and n <= b. Up to you, you could send another PR if you wanted to. :)